components/sensor/sht3xd/sht3xd.go

```diff
@@ -0,0 +1,182 @@
+// Package sht3xd implements a sht3x-d sensor for temperature and humidity
+package sht3xd
```

Comment:
[minor] add datasheet link, and any relevant repos we should be looking at if ever troubleshooting this component

could find [this](https://cdn-shop.adafruit.com/product-files/2857/Sensirion_Humidity_SHT3x_Datasheet_digital-767294.pdf), but want to confirm that was what you were looking at before adding to the rdk drive.